source/includes/rankings.md

```diff
@@ -165,6 +165,23 @@ curl_close($ch);
 ```
 
 Request all rankings by date and by country. This call returns ranking records with all the available meta-data from the store's ranking list page.
+Currently the following countries are supported:
```

Comment:
Change to: The following countries are supported

Reply:
Done.

Reply:
https://github.com/AppMonsta/slate/commit/6a6e795fd366016d1a09c83c5dec646f981324c3